### ISSUE TITLE
Uncaught Error: application 'node' died in status SKIP_BECAUSE_BROKEN:[$injector:modulerr]

### ISSUE THREAD

**Issue by MimoJSRepo** [single-spa/single-spa-angularjs#58]
I am getting an error message:

`Uncaught Error: application 'something' died in status SKIP_BECAUSE_BROKEN:[$injector:modulerr]`

```js
var singleApp = window.singleSpaAngularjs.default({
    angular: window.angular,
    domElementGetter: function() {
        return document.getElementById('something')
    },
    mainAngularModule: 'app',
    uiRouter: true,
    preserveGlobal: true,
    // template: '<any />',
})

window.singleSpa.registerApplication(
    'something',
    singleApp,
    function activityFunction() {
        return true;
    },
);

window.singleSpa.start();
```

does anyone have any idea what this could be?

plus, there is no much content or examples that cover angularjs applications that use angular-ui-router, so it is a little bit hard to follow, since that the new version 5 documentation  is not giving a clear example

**Comment by joeldenning** [comment]
This error likely happened when the appllication was mounting. Single-spa-angularjs calls `angular.bootstrap()`, which kicks off all dependency injection and mounts to the DOM.

The more important information is to see the cause for the `$injector:modulerr`. It's likely due to an angularjs module not being available for injection.

A full stacktrace would help in diagnosing, but a demonstration of the issue would help even more. I'm closing this as there's no demonstration.
